x-team/xp-ui
# Make the header bar appear above all elements on the profile page

**Release Type:** *Bug Fix* 

Fixes https://x-team-internal.atlassian.net/browse/XP-3657

## Description

Set the z-index to the HeaderBar component to 1000. This is greater than
every UI component that appears on that page.

Remove the z-index from the element that contains the HeaderBar on the
ApplicantInfo page. Setting a z-index on that element creates a new
"stacking context" in CSS which makes the z-index for contained
elements do the wrong thing.

## Checklist


## Steps to Test or Reproduce

Visit `http://localhost:8001/applicant/:token/info` in your browser. Note that as you scroll down input elements appear above the bar at the top of the streen.

Build these changes, and copy the generated files in `lib/` into `xp-registration/site/node_modules/xp-ui/lib`.

Now refresh the `applicant/:token/info` page. This time you can scroll up and down and input elements never go above the top bar.

## Impacted Areas in Application

Applicant profile ("info") page.

## Screenshots

![scroll-z-index](https://user-images.githubusercontent.com/5854132/77421922-5b925600-6dd5-11ea-80ee-66cea747a8f4.gif)
